src/defines.cpp

```diff
@@ -1,117 +0,0 @@
-/**
- * @file defines.cpp
- * Manages the table of defines for some future time when these will be used to
- * help decide whether a block of #if'd code should be formatted.
- *
- * !! This isn't used right now. !!
```

Comment:
Well, this comment would seem to agree with your belief :slightly_smiling_face:.